dspace-api/src/test/java/org/dspace/app/util/GoogleBitstreamComparatorTest.java

```diff
@@ -92,9 +92,9 @@ public void testPDFDifferentSize() throws Exception {
         when(bitstreamFormat1.getMIMEType()).thenReturn("application/pdf");
         when(bitstreamFormat2.getMIMEType()).thenReturn("application/pdf");
         when(bitstreamFormat3.getMIMEType()).thenReturn("application/pdf");
-        when(bitstream1.getSize()).thenReturn(new Long(100));
-        when(bitstream2.getSize()).thenReturn(new Long(200));
-        when(bitstream3.getSize()).thenReturn(new Long(300));
+        when(bitstream1.getSize()).thenReturn(Long.valueOf(100));
```

Comment:
Since the [ErrorProne checker was merged](https://github.com/DSpace/DSpace/pull/1861), you get a lot of warnings when compiling an individual test. These changes are here to solve some of these warnings and reduce the clutter.

Reply:
These warnings are also visible in Travis (on other PR's), e.g. https://travis-ci.org/DSpace/DSpace/builds/295357886#L3293

Reply:
Specifically, "new Integer(small value)" is inefficient, since Integer caches some instances of small values and will just return references instead of creating duplicates.  Many static analysis tools and IDEs will complain about this.  The same goes for all wrapped primitives and especially Boolean.

But why were bitstream2 and bitstream3 not treated?

Reply:
@mwoodiupui I guess you was dupe but my inline comments. Also the following use of new Long(...) has been changed by @tomdesair